.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: node_js
 node_js:
 - '6'
-before_install:
-- openssl aes-256-cbc -K $encrypted_369f74a373d5_key -iv $encrypted_369f74a373d5_iv
-  -in secrets.tar.enc -out secrets.tar -d
-- tar xvf secrets.tar
+before_install: "if [[ \"$TRAVIS_PULL_REQUEST\" = \"false\" ]]; then openssl aes-256-cbc -K $encrypted_369f74a373d5_key -iv $encrypted_369f74a373d5_iv -in secrets.tar.enc -out secrets.tar -d ; tar xvf secrets.tar ; fi"
```

Comment:
Ah - this looks useful. Could you raise a separate PR with that fix in it? Thanks!